### DISCUSSION TITLE
Translate Documents

### DISCUSSION THREAD

**Discussion by CyberSardinha** [jonaswinkler/paperless-ng#1410] [Feature Requests]
I think it would be a great feature if we could translate documents. Paperless could use the Google Translator engine to do it and take the text from the Content tab and get another tab for example Translated and get the text there.

I think it would be a great diffrentiating feature for this project.

We could use our own API key we get 500.000 characters for free.
Or there are a couple of libraries that try to bypass that but it looks like they need constant updates and probably not ideal for the project

**Comment by frank-from-cologne** [comment]
Hey, I am also interested in a translation feature.

What can we do to boost this?

Is it possible and customary to look for developers here who might implement this? 

If so, what is the best way to do this? Also with regard to the question of tendering/funding?

**Comment by CyberSardinha** [comment]
I don’t know.
I’m quite happy to contribute to a developer willing to implement this.

**Comment by frank-from-cologne** [comment]
… just to keep this alive … 😉
